### ISSUE TITLE
Attach output should give clear instructions when application is already started

### ISSUE THREAD

**Issue by mehmetf** [flutter/flutter#32905]
The attach output (if not run with debug-uri or url) currently says: 

```
Waiting for a connection from Flutter on Pixel 3 XL...
```

which does not prompt for a user action.

We should likely enrich this to say:

```
If your app is already running, please restart it (or pass --debug-uri <observatory_url>)
Waiting for your app to run on Pixel 3 XL...
```

**Comment by cv692001** [comment]
Hey , I am new to contributing can you please help me that how should i proceed ?

**Comment by jmagman** [comment]
@mehmetf #53153 allows the tool to back-read the Android logs, does that cover this request?

**Comment by mehmetf** [comment]
Yes. In most cases, if you are now looking for the last known observatory url in the old logs, this will work.

What happens if you can't find the log entry though? I assume that will happen if they have not run the app yet. Do you fail or still show message "Waiting for connection..." ?

Should we always require app to have started before running attach? That's normally how attach works on other systems (like attaching to JVM).
